localtypings/pxtpackage.d.ts

```diff
@@ -161,4 +162,39 @@ declare namespace pxt {
         namespace?: string; // used to construct id
         mimeType: string;
     }
+
+    interface SnippetConfig {
+        name: string;
+        namespace: string;
+        group?: string;
+        label: string;
```

Comment:
The CLI needs to traverse the config tree and extract the strings that need to be translated into the target-strings.json file. Then you need to use rlf in the UI to localize the strings.